### PR TITLE
Install Mizu in a dedicated namespace

### PR DESCRIPTION
**Pull request by nimrod-up9**
1. Install Mizu resources (tappers daemonset, controller pod, collector service, serviceacount) in a dedicated "mizu" namespace.
2. When removing Mizu resources, delete the namespace instead of deleting each resource individually.
3. When removing Mizu resources, remove clusterrole and clusterrolebinding.
4. When removing Mizu resources, wait for resources to be deleted before exiting.
5. A second ctrl-c will skip the wait.

### CODE REVIEW

File: cli/mizu/consts.go [hunk 1]
```diff
@@ -9,11 +9,14 @@ var (
 )
 
 const (
-	ResourcesNamespace  = "default"
-	TapperDaemonSetName = "mizu-tapper-daemon-set"
-	AggregatorPodName   = "mizu-collector"
-	TapperPodName       = "mizu-tapper"
-	K8sAllNamespaces    = ""
+	AggregatorPodName      = "mizu-collector"
```
Comment (IgorGov) [inline cli/mizu/consts.go:12]: CollectorPodName

File: cli/kubernetes/provider.go [hunk 2]
```diff
@@ -71,7 +113,16 @@ func (provider *Provider) GetPodWatcher(ctx context.Context, namespace string) w
...
+func (provider *Provider) CreateNamespace(ctx context.Context, name string) (*core.Namespace, error) {
+	namespaceSpec := &core.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		},
+	}
+	return provider.clientSet.CoreV1().Namespaces().Create(ctx, namespaceSpec, metav1.CreateOptions{})
+}
+
+func (provider *Provider) CreateMizuAggregatorPod(ctx context.Context, namespace string, podName string, podImage string, serviceAccountName string, mizuApiFilteringOptions *shared.TrafficFilteringOptions) (*core.Pod, error) {
```
Comment (IgorGov) [inline cli/kubernetes/provider.go:125]: Aggregator => Collector

**Review by RamiBerm — DISMISSED** [review]

**Comment by ksudhir007** [comment]
Thank you very much for this! Can you also please update README permissions sections as part of this pull request ? It will help others give an overview on the RBAC permissions in shared cluster environment.

**Comment by nimrod-up9** [comment]
> Thank you very much for this! Can you also please update README permissions sections as part of this pull request ? It will help others give an overview on the RBAC permissions in shared cluster environment.

Good idea! Updated the README and added example resource yamls for the required clusterroles.

**Review by IgorGov — APPROVED** [review]

**Pull request merged**